src/pdf/FontAwesomeIconPdfComponent.tsx

```diff
@@ -4,17 +4,24 @@ import { ReactElement } from "react";
 
 interface FontAwesomeIconProps {
     faIcon: IconDefinition;
+    color?: string;
+    styleWidth?: string;
+    marginTop?: string;
 }
 
-const FontAwesomeIconPdfComponent = ({ faIcon: { icon } }: FontAwesomeIconProps): ReactElement => {
+const FontAwesomeIconPdfComponent = ({
+    faIcon: { icon },
+    color = "black",
+    styleWidth = "20px",
+    marginTop = "0px",
+}: FontAwesomeIconProps): ReactElement => {
```

Comment:
Added some more optional parameters with the defaults set to reflect what was already hard-coded (so that the other icons are not affected)
styleWidth to make the congestion charge logo smaller, and marginTop to give a bit of margin above it now that it floats to the top instead of being centred